src/internal/common/ptr/pointer.go

```diff
@@ -43,3 +43,7 @@ func OrNow(t *time.Time) time.Time {
 
 	return *t
 }
+
+func To[T any](t T) *T {
```

Comment:
nit: This does not seem to be used anywhere.

Reply:
removed